### PR TITLE
Major merge updating all ontologies

### PR DESCRIPTION
**Pull request by fbastian**
@cmungall Could you please check that you're happy with `src/ssso-merged.obo` and `src/ssso-merged-uberon.obo` and accept the merge request? All major ontologies have been updated (e.g., HsapDv, MmusDv), and all external ontologies have been updated to the latest version (ZFS, FBdv, WBls, XAO (dev. stages only)).

### CODE REVIEW

File: src/acardv/acardv-bridge-to-uberon.obo [hunk 1]
```diff
@@ -1,49 +1,79 @@
...
 intersection_of: UBERON:0000112
 intersection_of: part_of NCBITaxon:28377
 
 [Term]
-id: AcarDv:0000000
-property_value: IAO:0000589 "lizard life cycle stage (anolis)" xsd:string
-relationship: only_in_taxon NCBITaxon:28377 ! anolis
-intersection_of: UBERON:0000105
-intersection_of: part_of NCBITaxon:28377
+id: UBERON:0000105
```
Comment (cmungall) [inline src/acardv/acardv-bridge-to-uberon.obo:44]: something has gone wrong here,  UBERON:0000105 is "life cycle stage"

**Review by cmungall — COMMENTED** [review]
I don't think I will have time to review today...

File: src/acardv/acardv-bridge-to-uberon.obo (hunk 1 above)
Comment (fbastian) [reply src/acardv/acardv-bridge-to-uberon.obo:44]: I guess you have checked an erroneous commit, the final version https://github.com/obophenotype/developmental-stage-ontologies/blob/develop/src/acardv/acardv-bridge-to-uberon.obo has:

```
[Term]
id: AcarDv:0000000
property_value: IAO:0000589 "lizard life cycle stage (anolis)" xsd:string
intersection_of: UBERON:0000105
intersection_of: part_of NCBITaxon:28377
```

**Review by cmungall — APPROVED** [review]

**Review by cmungall — APPROVED** [review]
This can be merged

**Pull request merged**